Report/Sections/Phase-4/Evalution.tex

```diff
@@ -0,0 +1,79 @@
+                        & ZARA2 & 0.41 / 0.64  & 0.42 / 0.64 \\
+
+  \bottomrule
+  \end{tabular}
+  \label{table:adefde}
+\end{table}
+\hfill \\
+\begin{figure}[b]
+  \centering
+  \includegraphics[width=0.3\textwidth]{figures/disc_interpolations_code0_batch2.jpeg}
+  \caption{latent traversals of three dimensional categorical code for model UNIV\_12}{The x-axis is the traversals of three dimensions, The y-axis is the different samples. The black dots show the observed trajectory, the green dots show the ground true of the next 12 steps trajectory, the red dots show the prediction of our model}
+  \Description{latent traversals disc code}
+  \label{disc_code}
+\end{figure}
+
+\begin{figure*}[ht]
+  \centering
+  \includegraphics[width=0.8\textwidth]{figures/cont_interpolations_code0_batch2.jpeg}
+  \caption{latent traversals of the continuous latent code for model UNIV\_12}{The x-axis is the traversals of range $(-2, 2)$, The y-axis is the different samples. The black dots show the observed trajectory, the green dots show the ground true of the next 12 steps trajectory, the red dots show the prediction of our model}
+  \Description{latent traversals disc code}
+  \label{cont_code}
+\end{figure*}
+
+We first evaluate the performance of the modified model and compare it with other models. Table~\ref{table:adefde} shows the results of our modified model under one categorical latent code and one continuous latent codes. We trained our model with the following hyperparameters: one categorical latent code with three dimensions, one continuous latent code, the weight of infomation loss range from $0.03$ to $0.1$. The other hyperparameters are the same as the hyperparameters of SGAN model we trained. It can be seen that our results are slightly better than the SGAN model. This may be due to the fact that the additional InfoGAN slightly helps to structure the latent space.
+
+
+To observe the performance of untangling, we plotted the latent traversal for different samples. Figure~\ref{disc_code} shows the latent traversal of the categorical latent code. It is be noted that the interpretation of the discrete codes is difficult. This is not surprising, probably because it is not easy to describe categorical information in a two-dimensional coordinate space.
+
+Figure~\ref{cont_code} shows the traversals of the continuous code. It can be seen that there is a roughly consistent variation in the continuous latent codes across most of the samples. (e.g. move the trajectory to downside. But that is not always the case, with the code increase, the direction of the trajectory slightly change). We must point out that the information loss for the continuous code cannot decrease to a low level, as shown in Figure. This indicates that the information between the latent code and the generated samples has not been maximized, so the latent space is still entangled. This might also explains the fact that our InfoGAN has only a slight improvement in ADE/FDE.
```

Comment:
"move the trajectory to downside", maybe it's better to say "move the trajectory downwards"